draft-privacy-token.md

```diff
@@ -58,23 +62,25 @@ of the structure contents.
 ~~~
 struct {
     uint8_t version;
-    uint8_t key_id[4];
+    uint8_t key_id[8];
     uint8_t message[32];
-    uint8_t signature[512];
+    uint8_t signature[Nk];
 } Token;
 ~~~
 
 The structure fields are defined as follows:
 
 - "version" is a 1-octet integer. This document defines version 1.
 
-- "key_id" is a 4-octet truncated key ID that identifies the key used to produce
-the signature. This is generated as SHA256(public key)[0:32].
+- "key_id" is a 8-octet truncated key ID that identifies the key used to produce
```

Comment:
I thought we had agreed to 4 bytes?

Reply:
Ooooh, I see. Now the request on the other issue.

8 bytes is still feasible to find a collision.

I'm discussing using the same blind signature scheme for another protocol where the KeyID forgery would break the security of the scheme.

Is there any opposition to just not truncate the hash?

Reply:
None from me, but see my comment below where I propose the removing the need for clients to compute this at all.